appengine/standard/firebase/firenotes/frontend/main.js

```diff
@@ -19,15 +19,19 @@ $(function(){
   // backend's app.yaml file.
   var backendHostUrl = '<your-backend-url>';
 
+  // [START config]
```

Comment:
My understanding is that now that we have the sample tracker, we need to use tag titles that are non-ambiguous even across files. Perhaps something like [START gae_firenotes_config]?

Reply:
Done!